template.distillery/README.md

```diff
@@ -222,3 +228,24 @@ you update the server:
 ```
 make APP_SERVER=http://${YOUR_SERVER} APP_REMOTE={yes|no} chcp
 ```
+
+## Use Makefile.local file.
+
+You need to define `APP_REMOTE` and `APP_SERVER` each time you want to build
+mobile applications or to update it.
+
+If you don't want to pass every time the variables `APP_SERVER` and
+`APP_REMOTE`, you can change the values of these variables in
```

Comment:
_pass the variables `APP_SERVER` and `APP_REMOTE` every time_ (move _every time_)